.drone.yml

```diff
@@ -232,10 +232,11 @@ steps:
           - master
           - feature/*
       event: pull_request
-
+# Anchore To be replaced with trivy.
```

Comment:
keep the blank line before for readability

Reply:
done